sor-api/src/main/java/com/bazaarvoice/emodb/sor/api/Names.java

```diff
@@ -19,4 +27,18 @@ public static boolean isLegalTableAttributeName(String attributeName) {
         // The attributes should not start with "~" which is reserved for Emodb's internal use
         return !attributeName.startsWith("~");
     }
+
+    /**
+     * Queue names must be lowercase ASCII strings. between 1 and 255 characters in length.  Whitespace, ISO control
+     * characters and most punctuation characters aren't allowed.  Queue names may not begin with a single underscore
+     * to allow URL space for extensions such as "/_queue/...".  Queue names may not look like relative paths, ie.
+     * "." or "..".
+     */
+    public static boolean isLegalQueueName(String queue) {
```

Comment:
question: where are we using this function?

Reply:
unused code. Removed.